tests/dragonfly/json_test.py

```diff
@@ -84,3 +84,30 @@ async def test_update_value(async_client: aioredis.Redis):
     except redis.exceptions.ResponseError as e:
         assert e.args[0] == "WRONGTYPE Operation against a key holding the wrong kind of value"
     assert await async_client.type(key_name) == "string"
+
+
+@pytest.mark.parametrize(
+    "description,expected_value,expected_type",
+    (
+        ("array", "[]", "array"),
+        ("string", dumps("dragonfly"), "string"),
+        ("number", dumps(3.50), "number"),
+        ("object", dumps({"dragon": "fly"}, separators=(",", ":")), "object"),
+        ("boolean true", "true", "boolean"),
+        ("boolean false", "false", "boolean"),
+    ),
+)
+@pytest.mark.asyncio
+async def test_arrappend(async_client: aioredis.Redis, description, expected_value, expected_type):
+    key_name = "test-json-key"
+
+    await async_client.execute_command("json.set", key_name, "$", "[]")
+    await async_client.execute_command("json.arrappend", key_name, "$", expected_value)
+
+    # make sure the value is as expected
+    first_element = await async_client.execute_command("json.get", key_name, "$[0]")
+    assert first_element == "[{}]".format(expected_value)
+
+    # make sure the type is as expected
+    actual_type = await async_client.execute_command("json.type", key_name, "$[0]")
+    assert actual_type[0] == expected_type
```

Comment:
Great you covered it with extensive tests 👍🏻 

You can execute json commands directly with the python client, e.g. `async_client.json().arrappend()`. However I've just noticed our json tests don't use it as well 🤯  So I don't expect you either 🙂